source/extensibility/syntaxdefs.rst

```diff
@@ -163,7 +163,7 @@ Creating A New Syntax Definition
 
 To create a new syntax definition, follow these steps:
 
-  - Go to **Tools | Packages | Package Development | New Syntax Definition**
+  - Go to **Tools | Developer | New Syntax...**

```

Comment:
While the change isn't wrong per se, it still causes the following section to be misleading because that still targets the old tmLanguage format (or its YAML variant) instead of the new sublime-syntax format. For consistency with the current version of this document, this should refer to the "PackageDev: New TextMate Syntax Definition (YAML)" command in the palette.

At least until we make a new one for the new format … eventually …